content/1.7/back-office/catalog/discount/add-edit-cart-price-rule.md

```diff
@@ -0,0 +1,84 @@
+- **Number** of products required in the cart to enjoy the discount - defines the oparating value, how many products should be added to the cart in Prestashop application in order to get the ability to get a discount. There is a numeric value input field near, which can be set.
+- **Add a rule concerning** - defines the operating value, which will be included in creating the rule conditions for certain product (s). By default the dropdown option will be as null value --Choose--. The default values, that are installed by Prestashop are Products, Attributes, Categories, Brands and Suppliers. Choosing, for example, Products, as the value, clicking the (+) Add button will extend the UI with the Product list below. By default, there will be no products added. Once clicking the Choose button, pop-up will appear, and there will be a possibility to add the certain products from left section to the right and backwards. It can be adjusted by clicking the Add or Remove buttons. After closing the popup, the new products will be added to the list. There are options to add additional value once again, modify the existing one or deleting the value from the list.
+
+- **Shop selection** - this selection sets the shop in Prestashop, where to apply the Cart Rule. By default, this field is hidden, if there is no Multistores configured. Once the Shop selection checkbox is marked, the addition UI part will extend with the 2 input area fields. There will be field with Selected shops and Unselected shops. The Multistore Shops can be transferred to the Selected or Unselected fields by appropriate needs. Add or Remove buttons are performing the actioins here.
+
+## Actions tab
+The certain action can be set after cart rule is created. The Actions tab values are all set up to disabled by default.
+
+![Cart Rules: Actions Tab](/static/img/cart-rules-actions-tab.png)
+
+- **Free shipping switch toggle button** - this setting creates rule that will initiate a free shipping method for customer in checkout form.
+- **Apply a discount** - there are 3 options that can be set - Percentage, Amount and None. Percentage can be set, and the cart price will be affected by certain amount of percentage discount, after executing the voucher. When the option is set with the certain Amount - the user will get a discount of a certain amount of price. Also the Currency can be set and the Taxes conditions as well. This option can set the action to None - in that case user of the checkout will not get any discounting at all.
+- **Send a free gift switch toggle button** - this option enables ability to set an additional free product for the customer, or every customers, who accepted the cart rule / voucher.
+- **Search a product field** - it will set the ability to search the certain product.
+- **Matching products dropdown** - it will appear when there will be some characters typed in the "Search a product" field.
+
```

Comment:
I think the "Apply the discount to" is missing.

And then, we should say that if the reduction apply to a specific product, then the search a product field appears : 
- **Search a product field** - it will set the ability to search the certain product.
- **Matching products dropdown** - it will appear when there will be some characters typed in the "Search a product" field.

This one should be listed last:
- **Send a free gift switch toggle button** - this option enables ability to set an additional free product for the customer, or every customers, who accepted the cart rule / voucher.